justfile

```diff
@@ -63,3 +63,14 @@ workspace-optimize:
 		--mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
 		--platform linux/amd64 \
 		cosmwasm/workspace-optimizer:0.12.11
+
+workspace-optimize-arm:
+    docker run --rm -v "$(pwd)":/code \
+        --mount type=volume,source="$(basename "$(pwd)")_cache",target=/code/target \
+        --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
+        --platform linux/arm64 \
+        cosmwasm/workspace-optimizer-arm64:0.12.11
+    # rename the wasm files so that they work with cosm-orc
+    # integration test contract names (which are expected to be the
+    # contract name, without the aarch64 postfix).
+    for file in artifacts/*-aarch64.wasm; do mv -- "$file" "${file%-aarch64.wasm}.wasm"; done
```

Comment:
I tried to handle this situation here:  https://github.com/de-husk/cosm-orc/blob/main/src/orchestrator/cosm_orc.rs#L153-L157

I wonder why it's not working in this case?

Reply:
possible it was failing for unrelated reasons when i added this. i'll test before merging.

Reply:
@de-husk confirmed that your solution works and my change was not needed. thanks for pointing this out!